src/utils/ContainerUtils.ts

```diff
@@ -0,0 +1,89 @@
+import { ContainerProps, ListView } from "../components/CheckBoxFilterContainer";
+
+export const parseStyle = (style = ""): {[key: string]: string} => {
+    try {
+        return style.split(";").reduce<{[key: string]: string}>((styleObject, line) => {
+            const pair = line.split(":");
+            if (pair.length === 2) {
+                const name = pair[0].trim().replace(/(-.)/g, match => match[1].toUpperCase());
+                styleObject[name] = pair[1].trim();
+            }
+            return styleObject;
+        }, {});
+    } catch (error) {
+        // tslint:disable-next-line no-console
+        window.console.log("Failed to parse style", style, error);
+    }
+
+    return {};
+};
+
+export class Utils {
+    static validate(props: ContainerProps & { filterNode: HTMLElement; targetListView: ListView; isModeler?: boolean}): string {
+        const widgetName = props.friendlyId;
+        const { targetListView } = props;
+        const errorMessage = [];
+        // validate filter values if filterby = attribute, then value should not be empty or "" or " ".
```

Comment:
Comments should stat with a capital (It like a normal English sentence)
`// Validate`